Source/ReadmeBuilder.cs

```diff
@@ -93,86 +103,71 @@ public void SetupHeader(ModelGroup test)
             readme.Add(firstLine);
             readme.Add(secondLine);
 
-            for (int i = 0; i < test.Properties.Count; i++)
+            // Generates the list of column names for use when setting up the table, and generates that part of the table as well.
+            foreach (var property in test.Properties)
             {
                 string attributeName;
-                attributeName = test.Properties[i].ReadmeColumnName;
+                attributeName = property.ReadmeColumnName;
 
-                if (attributeName != lastName) // Skip duplicate names caused by non-binary attributes
+                if (!columnNames.Contains(property.Name))
                 {
-                    lastName = attributeName;
                     readme[1].Add(attributeName);
                     readme[2].Add(":---:");
+                    columnNames.Add(property.Name);
                 }
             }
         }
 
+        /// <summary>
+        /// Builds the strings used to make the main table for each model group's readme.
+        /// </summary>
+        /// <param name="test"></param>
+        /// <param name="modelIndex"></param>
+        /// <param name="model"></param>
         public void SetupTable(ModelGroup test, int modelIndex, List<Property> model)
         {
             string modelGroupName = test.Name.ToString();
             string modelNumber = modelIndex.ToString("D2");
-            string liveURL = string.Format("https://bghgary.github.io/glTF-Assets-Viewer/?folder={0}&model={1}",
-                test.Id, modelIndex);
+            string liveURL = string.Format("https://bghgary.github.io/glTF-Assets-Viewer/?folder={0}&model={1}", test.Id, modelIndex);
```

Comment:
Consider using [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) everyone instead of string.Format

Reply:
Sounds like a good idea. I think this should be done as part of a PR focused on cleaning up `ReadmeBuilder` and `ReadmeStringBuilder`

Reply:
Sure.